### PR TITLE
Dts 36462 squad capacity

### PR DESCRIPTION
**Pull request by Chittauri**

### CODE REVIEW

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/scrum/service/CapacityServiceImpl.java [hunk 1]
```diff
@@ -67,19 +79,63 @@ public Map<String, Object> fetchKPIDataFromDb(Node leafNode, String startDate, S
...
 					basicProjectConfigId);
-			resultListMap.put(CAPACITY_DATA, capacityKpiData);
+
+			resultListMap.put(CAPACITY_DATA, getCapacityDataForAdditionalFilter(kpiRequest, capacityKpiData));
 
 		}
 		return resultListMap;
 	}
 
+	private CapacityKpiData getCapacityDataForAdditionalFilter(KpiRequest kpiRequest, CapacityKpiData capacityKpiData) {
```
Comment (purushottam08) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/scrum/service/CapacityServiceImpl.java:94]: add jocs

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/capacity/service/CapacityMasterServiceImpl.java [hunk 2]
```diff
@@ -573,4 +623,61 @@ public void deleteCapacityByProject(boolean isKanban, ObjectId basicProjectConfi
 		}
 	}
 
+	public void helper(CapacityMaster capacityMaster) {
```
Comment (purushottam08) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/capacity/service/CapacityMasterServiceImpl.java:626]: add docs

File: common/src/main/java/com/publicissapient/kpidashboard/common/model/application/AdditionalFilterCapacity.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ * limitations under the License.
+ *
+ ******************************************************************************/
+package com.publicissapient.kpidashboard.common.model.application;
+
+import java.util.List;
+
+import lombok.Data;
+
+@Data
```
Comment (purushottam08) [inline common/src/main/java/com/publicissapient/kpidashboard/common/model/application/AdditionalFilterCapacity.java:24]: add author in both newly added model classes

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/KpiDataHelper.java [hunk 4]
```diff
@@ -129,6 +129,26 @@ public static String createAdditionalFilterMap(KpiRequest kpiRequest, Map<String
 		return subGroupCategory;
 	}
 
+	public static void createAdditionalFilterMapForCapacity(KpiRequest kpiRequest, Map<String, Object> mapOfFilters,
```
Comment (purushottam08) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/KpiDataHelper.java:132]: add doc

**Review by purushottam08 — APPROVED** [review]

**Review by HinPublicis — APPROVED** [review]

**Review by brahmanand1 — APPROVED** [review]

**Pull request merged**